3-4-1/compile_tables.R

```diff
@@ -0,0 +1,44 @@
+# Author: Emma Wood
+# Initial date: 16/12/2021
+# purpose: set up for 3-4-1 indicator update, and save output files
+#     This file is directly called by update_indicator_main.R.
+#     It has to be named compile_tables.R
+#     It is the control script that runs all the others.
+
+library('stringr')
+library("dplyr")
+
+setwd("3-4-1") # this line is to run the template only - 
+# do not copy into your code as this is usually found in update_indicator_main.R
+
+source("example_config.R") # pulls in all the configurations
+source("compile_tables.R") # does the donkey-work of making the csv
+# at this point you should see lots of variables appear in the global environment 
+# pane (top right). These have been created by the update_type_1 script.
+
+existing_files <- list.files()
+output_folder_exists <- ifelse(output_folder %in% existing_files, TRUE, FALSE)
+
```

Comment:
add a loop in here to go through the years. e.g.
```
# run one year of data at a time as the dataset is too big to download all years 
# at once
csv <- NULL

for (i in 1:length(years)) {
  all_but_female_link_temp <- paste0(all_but_female_link, "&date=", years[i])
  female_link_temp <- paste0(female_link, "&date=", years[i])
  
  source("update_3-4-1.R") 
  
  csv <- bind_rows(csv, csv_output)
}
```

Reply:
Yes, this makes sense. Done.